### ISSUE TITLE
do not copy `target` folder (which is huge and thus slow)

### ISSUE THREAD

**Issue by fzyzcjy** [mityax/rustimport#9]
Hi thanks for the lib! My usage (on my local dev computer) is as follows: 

1. run `cargo test` etc
2. also use rustimport and thus use in python code

However, it seems that the rustimport tries to copy all folders, including the huge `target` folder of my workspace, into the temporary dir. Since the target folder is more than a few GB, it is slow to copy and wastes disk and time.

PS. A stack trace when I realize rustimport is stuck and hit keyboard interrupt is as follows. As can be seen, it seems to be copying.

```
^CTraceback (most recent call last):
  File "/Users/tom/opt/anaconda3/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Users/tom/opt/anaconda3/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/tom/Main/research/code/research_mono/research_mono/benchmarks/benchmark_ineq_generator.py", line 17, in <module>
    main()
  File "/Users/tom/Main/research/code/research_mono/research_mono/benchmarks/benchmark_ineq_generator.py", line 9, in main
    g = wrapped_generator.GeneratorVec(hypotheses=hypotheses, num_generators=num_generators)
  File "/Users/tom/Main/research/code/research_mono/research_mono/neural_theorem_prover/interactive_theorem_prover/ineq/generator/wrapped_generator.py", line 8, in __init__
    import rust.ineq
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 664, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 627, in _load_backward_compatible
  File "/Users/tom/opt/anaconda3/lib/python3.9/site-packages/rustimport/import_hook.py", line 48, in load_module
    self.__importable.build(release=settings.compile_release_binaries)
  File "/Users/tom/opt/anaconda3/lib/python3.9/site-packages/rustimport/importable.py", line 206, in build
    shutil.copytree(self.__workspace_path or self.__crate_path, root_output_path, dirs_exist_ok=True)
  File "/Users/tom/opt/anaconda3/lib/python3.9/shutil.py", line 566, in copytree
    return _copytree(entries=entries, src=src, dst=dst, symlinks=symlinks,
  File "/Users/tom/opt/anaconda3/lib/python3.9/shutil.py", line 504, in _copytree
    copytree(srcobj, dstname, symlinks, ignore, copy_function,
  File "/Users/tom/opt/anaconda3/lib/python3.9/shutil.py", line 566, in copytree
    return _copytree(entries=entries, src=src, dst=dst, symlinks=symlinks,
  File "/Users/tom/opt/anaconda3/lib/python3.9/shutil.py", line 504, in _copytree
    copytree(srcobj, dstname, symlinks, ignore, copy_function,
  File "/Users/tom/opt/anaconda3/lib/python3.9/shutil.py", line 566, in copytree
    return _copytree(entries=entries, src=src, dst=dst, symlinks=symlinks,
  File "/Users/tom/opt/anaconda3/lib/python3.9/shutil.py", line 504, in _copytree
    copytree(srcobj, dstname, symlinks, ignore, copy_function,
  File "/Users/tom/opt/anaconda3/lib/python3.9/shutil.py", line 566, in copytree
    return _copytree(entries=entries, src=src, dst=dst, symlinks=symlinks,
  File "/Users/tom/opt/anaconda3/lib/python3.9/shutil.py", line 508, in _copytree
    copy_function(srcobj, dstname)
  File "/Users/tom/opt/anaconda3/lib/python3.9/shutil.py", line 444, in copy2
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/Users/tom/opt/anaconda3/lib/python3.9/shutil.py", line 270, in copyfile
    _fastcopy_fcopyfile(fsrc, fdst, posix._COPYFILE_DATA)
  File "/Users/tom/opt/anaconda3/lib/python3.9/shutil.py", line 105, in _fastcopy_fcopyfile
    posix._fcopyfile(infd, outfd, flags)
KeyboardInterrupt

**Comment by fzyzcjy** [comment]
Fixed by #10

**Comment by mityax** [comment]
Thanks for your improvement suggestions and PR to rustimport so far! I'm very happy everytime it gets a little more mature.

**Comment by fzyzcjy** [comment]
You are welcome! rustimport is quite good already ;)

**Comment by mityax** [comment]
Fixed now in [70f172b](https://github.com/mityax/rustimport/commit/70f172b1ccce695ae929b314fa8c95cd701b07d7)
